OracleWebLogic/samples/12212-msiserver/README.md

```diff
@@ -18,21 +18,14 @@ Next, to build the base msi image, run:
 
 Finally, to start the Managed Server in MSI mode, run:
 
-        $ docker run -d -p 8011:8011 12212-msiserver
+        $ docker run -p 8011:8011 12212-msiserver

```

Comment:
Why did you remove the `-d` here? The image needs to be run in daemon mode, i.e. the background You should then use `docker logs` to retrieve the `stdout` from the running container to determine things like the MS name.